org.musicbrainz.Picard.json

```diff
@@ -167,10 +235,10 @@
             "name": "python-libdiscid",
             "buildsystem": "simple",
             "build-commands": [
-                "python2 setup.py install --prefix=/app"
+                "python3 setup.py install --prefix=/app"
             ],
             "ensure-writable": [
-                "/lib/python2.7/site-packages/easy-install.pth"
+                "/lib/python3.5/site-packages/easy-install.pth"
```

Comment:
Using Pip to install things with Python 3, this becomes unnecessary and can be removed entirely. :slightly_smiling_face: